src/index.ts

```
@@ -1257,6 +1327,7 @@ export const pluginReactRouter = (
                       subResourceIntegrity:
                         resolvedConfigWithRoutes.subResourceIntegrity,
```

Biome (2.5.0)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)